manifests/manifests-maistra/maistra.package.yaml

```diff
@@ -0,0 +1,8 @@
+packageName: maistraoperator
+channels:
+- name: "1.0"
+  currentCSV: maistraoperator.v1.1.0
```

Comment:
shouldn't this point to `1.0.8`?

Reply:
Moving forward, we will only be maintaining a single operator, so the `1.1` operator will provide functionality for `1.0` installs.  Once the `1.1` operator is available, we should stop support on the `1.0` operator.